lambeq/backend/grammar.py

```diff
@@ -1912,14 +1970,14 @@ class Functor:
     >>> n = Ty('n')
     >>> diag = Cap(n, n.l) @ Id(n) >> Id(n) @ Cup(n.l, n)
     >>> diag.draw(
-    ...     figsize=(2, 2), path='./snake.png')
+    ...     figsize=(2, 2), path='./docs/_static/images/snake.png')
```

Comment:
Be sure you pull the main again, this file here looks outdated (or some problem happened with conflict resolution).

Reply:
Also, be sure to keep the upstream changes for these images above.